projects/storefrontlib/src/shared/components/list-navigation/search/search.component.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+} from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { ICON_TYPE } from '../../../../cms-components/misc/icon/icon.model';
+
+@Component({
+  selector: 'cx-search',
+  templateUrl: './search.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SearchComponent {
```

Comment:
I cannot find any usages of `cx-search` component in our code base anymore
@Pucek9 wouldn't you mind if we remove the search component with its styles and module?

Reply:
In our first wireframes, we had search functionality, but then it was verified that backend doesn't support that. I dont know the status of work on this. IMO we can remove it, we have backup branch if the topic comes back :)

Reply:
i'm dropping it